spyder/widgets/mixins.py

```diff
@@ -754,8 +754,12 @@ def add_to_history(self, command):
         if self.history_filename not in self.HISTORY_FILENAMES:
             self.HISTORY_FILENAMES.append(self.history_filename)
             text = self.SEPARATOR + text
-        
-        encoding.write(text, self.history_filename, mode='ab')
+        # Needed to prevent errors from the history.py

```

Comment:
Please change this comment to

    # Needed to prevent errors when writing history to disk